src/edu/csus/ecs/pc2/ui/SubmitSampleRunsPane.java

```diff
@@ -0,0 +1,877 @@
+
+		JPanel centerPane = new JPanel();
+		centerPane.setLayout(new BorderLayout());
+		add(centerPane, BorderLayout.CENTER);
+
+		JSplitPane splitPane = new JSplitPane();
+		splitPane.setOrientation(JSplitPane.VERTICAL_SPLIT);
+		centerPane.add(splitPane, BorderLayout.CENTER);
+
+		JPanel controlsPane = new JPanel();
+		controlsPane.setPreferredSize(new Dimension(400, 400));
+
+		controlsPane.setLayout(null);
+
+		JLabel cdpConfigDirLabel = new JLabel("CDP config dir");
+		cdpConfigDirLabel.setToolTipText("CDP Location for sample source files");
+		cdpConfigDirLabel.setHorizontalAlignment(SwingConstants.RIGHT);
+		cdpConfigDirLabel.setBounds(10, 60, 86, 14);
+		controlsPane.add(cdpConfigDirLabel);
+
+		cdpTextField = new JTextField();
+		cdpTextField.setFont(new Font("Tahoma", Font.PLAIN, 12));
+		cdpTextField.setBounds(113, 54, 418, 27);
+		controlsPane.add(cdpTextField);
+		cdpTextField.setColumns(10);
+		
+        cdpTextField.addKeyListener(new java.awt.event.KeyAdapter() {
+            public void keyPressed(java.awt.event.KeyEvent e) {
+                if (e.getKeyCode() == java.awt.event.KeyEvent.VK_ENTER) {
+                    if (new File(cdpTextField.getText()).isDirectory()) {
+                        updateCDPDirAndFields(cdpTextField.getText());
+                    }
+                }
+
+            }
+        });
+
+		messageLabel = new JLabel("message label");
+		messageLabel.setFont(new Font("Tahoma", Font.PLAIN, 14));
+		messageLabel.setForeground(Color.RED);
+		messageLabel.setHorizontalAlignment(SwingConstants.CENTER);
+		messageLabel.setBounds(10, 11, 691, 32);
+		controlsPane.add(messageLabel);
+
+		checkBoxSubmitYesSamples = new JCheckBox("Submit Yes Samples");
+		checkBoxSubmitYesSamples.setSelected(true);
+		checkBoxSubmitYesSamples.setToolTipText("Only submit AC sample source");
+		checkBoxSubmitYesSamples.setBounds(33, 102, 265, 23);
+		controlsPane.add(checkBoxSubmitYesSamples);
+
+		JButton selectCDPButton = new JButton("...");
+		selectCDPButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				selectNewCDP();
+			}
+		});
+		selectCDPButton.setToolTipText("Select a different CDP");
+		selectCDPButton.setBounds(541, 56, 39, 23);
+		controlsPane.add(selectCDPButton);
+
+		JPanel LogPanel = new JPanel();
+		LogPanel.setSize(new Dimension(400, 400));
+		LogPanel.setMinimumSize(new Dimension(400, 400));
+
+		LogPanel.setLayout(new BorderLayout(0, 0));
+
+		textArea = new JTextArea();
+		textArea.setFont(new Font("Courier New", Font.PLAIN, 13));
+		textArea.setSize(new Dimension(360, 360));
+		JScrollPane scrollPane = new JScrollPane(textArea);
+		LogPanel.add(scrollPane, BorderLayout.CENTER);
+
+		JPanel panel = new JPanel();
+		panel.setPreferredSize(new Dimension(50, 50));
+		FlowLayout flowLayout_1 = (FlowLayout) panel.getLayout();
+		flowLayout_1.setHgap(45);
+		LogPanel.add(panel, BorderLayout.SOUTH);
+
+		JButton clearTextAButton = new JButton("Clear");
```

Comment:
The Clear button should have a ToolTip explaining what it is going to cause to be "cleared".  (The user might think for example that it will clear all settings back to their default, which is not what it does.)